_author/fanny.md

```diff
@@ -0,0 +1,5 @@
+---
+fullname: Fanny Brûlebois
```

Comment:
Le système me refuse :-( snif 



> Le 18 mai 2016 à 19:04, Florian Delezenne <notifications@github.com> a écrit :
> 
> In _author/fanny.md:
> 
> > @@ -0,0 +1,5 @@
> > +---
> > +fullname: Fanny Brûlebois
> Je pense que le û ne passe pas, je ne sais pas exactement pourquoi
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>